packages/react-components/react-dialog/src/components/DialogSurface/useDialogSurfaceStyles.styles.ts

```diff
@@ -53,17 +49,14 @@ const useRootResetStyles = makeResetStyles({
   backgroundColor: tokens.colorNeutralBackground1,
   color: tokens.colorNeutralForeground1,
 
-  '&::backdrop': backdropStyles,
-
   [MEDIA_QUERY_BREAKPOINT_SELECTOR]: {
     maxWidth: '100vw',
   },
 });
 
 const useStyles = makeStyles({
-  nestedDialogBackdrop: nestedDialogBackdropStyles,
-  nestedNativeDialogBackdrop: {
-    '&::backdrop': nestedDialogBackdropStyles,
+  nestedDialogBackdrop: {
+    backgroundColor: 'transparent',
```

Comment:
Should it use `tokens.colorTransparentBackground`?

Reply:
great catch! I'll open a PR to resolve this for dialog!